clients/cloud/python/README.md

```diff
@@ -50,7 +50,7 @@ Successfully produced record to topic test1 partition [0] @ offset 9
 10 messages were successfully produced to topic test1!
 ```
 
-1. Run the consumer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the same topic name as used above. Verify that the consumer received all the messages:
+2. Run the consumer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the same topic name as used above. Verify that the consumer received all the messages:
 
 ```bash
 $ ./consumer.py -f ~/.ccloud/librdkafka.config -t test1
```

Comment:
we're changing this to example.config, right? (maybe in another PR?)

Reply:
I didn't for the Java example, it can (should?) use the one established by Confluent Cloud CLI?

Reply:
Yeah, the Java clients can/should use ~/.ccloud/config directly.
But since there is java-specific config in there it can't be reused for the other clients, it does help if all the other clients share the same config file though.

Reply:
Agree.  Non-java clients should use `~/.ccloud/example.config`